fix "typing blocked" bug

SimonThormeyer opened the pull request:
close #125

SimonThormeyer committed:
fix typing blocked-bug

close #125

chanty1512 reviewed (changes requested):
beim speicher overlay kann ich kein leerzeichen eingeben, stattdessen wird die looperaufnahme gestartet. 
wenn overlay erscheint sollte die tastenfunktion für die bedienung der loops nicht funktionieren, da es ja visuell extra ausgegraut ist. vielleicht bei overlay globalen state auf false setzen und bei handleKeyDown Function abfragen.

SimonThormeyer committed:
add overlayIsOpen to GlobalState / improve keyDown behaviour

event-handlers of keyDown Events will only be fired when overlayIsOpen is false

SimonThormeyer committed:
disable keyDown events for record overlay as well

chanty1512 reviewed (approved):
funktioniert wie es soll
